packages/react-components/react-charts-preview/src/components/CommonComponents/PopoverComponent.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';
+import { Popover, PopoverSurface } from '@fluentui/react-popover';
+import { mergeClasses, PositioningVirtualElement } from '../../../../react-components/src/index';
+import { ChartHoverCard, getAccessibleDataObject, Points, pointTypes } from '../../utilities/index';
+import { IChartDataPoint } from '../DonutChart/index';
+import { convertToLocaleString } from '../../utilities/locale-util';
+import { Shape } from '../Legends/shape';
+import { usePopoverStyles_unstable } from './PopoverComponent.styles';
+import { IYValueHover } from './CartesianChart.types';
+import { LegendShape } from '../Legends/Legends.types';
+import { IPopoverComponentProps } from './PopoverComponent.types';
+
+const PopoverComponent: React.FunctionComponent<IPopoverComponentProps> = React.forwardRef<
+  HTMLDivElement,
+  IPopoverComponentProps
+>((props, forwardedRef) => {
+  const virtualElement: PositioningVirtualElement = {
+    getBoundingClientRect: () => ({
+      top: props.clickPosition.y,
+      left: props.clickPosition.x,
+      right: props.clickPosition.x,
+      bottom: props.clickPosition.y,
+      x: props.clickPosition.x,
+      y: props.clickPosition.y,
+      width: 0,
+      height: 0,
+    }),
+  };
+
+  const classes = usePopoverStyles_unstable(props);
+
+  return (
+    <Popover positioning={{ target: virtualElement }} open={props.isPopoverOpen} inline>
+      <PopoverSurface tabIndex={-1}>
```

Comment:
need to add focusable group to keep the popover tabbable

Reply:
added in next PR